changelog/druntime.valgrind.dd

```diff
@@ -0,0 +1,42 @@
+Integration with the Valgrind `memcheck` tool has been added to the garbage collector
```

Comment:
Please add a clear and obvious note that use of Valgrind means complying with the Valgrind license, which is NOT the Boost license.

This means we cannot compile dmd itself with Valgrind.

Reply:
Not quite, the client API header files are under a different license than the rest of Valgrind. They are under a BSD-style license, which doesn't impose any restrictions that would affect us.

https://github.com/CyberShadow/dmd/blob/ae3e5d2731b422b20a538a7188061bd9e6624727/druntime/src/etc/valgrind/memcheck.h#L3-L56

Reply:
ok